app/domain/authentication/constraints/exclusive_constraint.rb

```diff
@@ -0,0 +1,24 @@
+require 'command_class'
+
+module Authentication
+  module Constraints
+
+    ExclusiveConstraint = CommandClass.new(
```

Comment:
please call command classes with the action it performs. the name `ExclusiveConstraint` indicates a value object